perception/sub8_perception/nodes/path_marker_finder.py

```diff
@@ -35,10 +35,18 @@ class PathMarkerFinder():
     TODO: Implement Kalman filter to smooth pose estimate / eliminate outliers
     """
     # Model of four corners of path marker, centered around 0 in meters
-    PATH_MARKER = np.array([[0.6096,  -0.0762, 0],
-                            [0.6096,  0.0762,  0],
-                            [-0.6096,  0.0762, 0],
-                            [-0.6096, -0.0762, 0]], dtype=np.float)
+    LENGTH = 0.6096 # Longer side of path marker in meters
```

Comment:
I kind of agree with Matt on not putting known constants as rosparams. These dimensions are known and in the rule book. Having them in YAML's might just be confusing when we are configuring things that may change.